File: virtual/round2/I.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
	// freopen ("important.in","r+",stdin);
	// freopen ("important.out","w+",stdout);
	int n;
	cin >> n;
	string A;
	char arr[n + 1];
	for(int i = 0; i < n; i++)
	{
		cin >> A;
		arr[i] = A[0];
	}
	cout << "Yes" << endl;
	int i;
	for(i = 1; i < n; i++)
	{
		cout <<  arr[i - 1] << "|" << "~" << arr[i - 1] << "|";
	}
	cout << arr[i - 1] << "|" << "~" << arr[i - 1] << endl;
	return 0;
}

File: virtual/round2/I.py
import sys
sys.stdin = open("important.in", 'r')
sys.stdout = open("important.out", 'w')
n = int(input())
print("Yes")
for i in range(n):
	a = input()
	if(i != n - 1):
		print(a[0] + "|~" + a[0] + "|", end ="")
	else:
		print(a[0] + "|~" + a[0])